apps/cnspec/cmd/scan.go

```diff
@@ -397,21 +395,28 @@ This example connects to Microsoft 365 using the PKCS #12 formatted certificate:
 		printReports(report, conf, cmd)
 
 		// handle report sharing
-		var shareReport *bool
+		var shareReport bool
```

Comment:
Should we rename this to 'web-report' for clarity?

Reply:
with the current assumption that we go with --share the current name is fine